lite2/client.go

```diff
@@ -84,11 +84,12 @@ func SkippingVerification(trustLevel tmmath.Fraction) Option {
 	}
 }
 
-// AlternativeSources option can be used to supply alternative providers, which
-// will be used for cross-checking the primary provider.
-func AlternativeSources(providers []provider.Provider) Option {
+// Witnesses option can be used to supply providers, which will be used for
+// cross-checking the primary provider. A witness can become a primary iff the
+// current primary is unavailable.
+func Witnesses(providers []provider.Provider) Option {
 	return func(c *Client) {
-		c.alternatives = providers
```

Comment:
Just as a thought. Do we want to check if these are on the same chain - i.e. `c.chainID == provider.ChainID()`. Also could be a way of initially checking if the provider is responding. Could be unnecessary thought.

Reply:
Good idea! We should do this upon startup. Want to do it? https://github.com/tendermint/tendermint/issues/4350